content/en/user-guide/snowflake-drivers/index.md

```diff
@@ -61,7 +61,7 @@ var connection = snowflake.createConnection({
     password: 'test',
     account: 'test',
     database: 'test',
-    host: 'snowflake.localhost.localstack.cloud',
+    accessUrl: 'https://snowflake.localhost.localstack.cloud',
```

Comment:
I think it would be good to have some pointer to the `host` config in our docs here, which seems to be the future-proof way of configuring the endpoint. Would it be worthwhile adding a note as a comment here, or even add both entries, `host` and `accessUrl`..?

Reply:
Good point, added a comment 👍